example-config.yaml

```diff
@@ -10,19 +10,69 @@ orgs:
       - name: junipernetworks.junos
       - name: cisco.asa
       - name: ansible.netcommon
+      - name: ansible.snmp
+      - name: ansible.scm
+      - name: ansible.security
+      - name: ansible.network
+      - name: frr.frr
+      - name: openvswitch.openvswitch
+      - name: community.yang
+      - name: ansible.utils
+      - name: ibm.qradar
+      - name: trendmicro.deepsec
+      - name: splunk.es
+    ci_repos:
```

Comment:
Please combine it to be a single config source, and make it generic enough to make the configuration not specific to ansible

Reply:
Sure will do it thanks

Reply:
Updated the tool to be more generic in the recent commit